src/Controllers/DatabaseController.js

```diff
@@ -1390,6 +1410,42 @@ class DatabaseController {
     }
   }
 
+  addProtectedFields(
```

Comment:
Confusing name to me.  This is returning an array of protected fields for  `auth` of this request..., right?

Reply:
That's correct, I was borrowing the naming convention from the previous function `addPointerPermissions`.